dspace-api/src/main/java/org/dspace/app/bulkedit/MetadataImport.java

```diff
@@ -688,6 +714,28 @@ private void handleRelationMetadata(Context c, Item item, String schema, String
 
     }
 
+    /**
+     * Gets an existing entity from a reference.
+     *
+     * @param context the context to use.
+     * @return the entity, which is guaranteed to exist.
+     * @throws MetadataImportException if the reference is badly formed or refers to a non-existing item.
+     */
+    private Entity getEntity(Context context, String reference) throws MetadataImportException {
```

Comment:
Can you include the reference in the JavaDoc, explaining what it expects (UUID or rowNum, …) similar to resolveEntityRef

Reply:
Improved related references docs and added to missing section.